Detections/MultipleDataSources/UserAgentSearch_log4j.yaml

```diff
@@ -77,15 +77,10 @@ query: |
   | summarize StartTime = min(TimeGenerated), EndTime = max(TimeGenerated) by UserAgent, SourceIP = IPAddress, Account = UserPrincipalName, Type, Operation = OperationName, tostring(LocationDetails), tostring(DeviceDetail), AppDisplayName, ClientAppUsed
   | extend timestamp = StartTime, AccountCustomEntity = Account, IPCustomEntity = SourceIP
   ),
-  (imWebSessions
+  (imWebSession
   | where HttpUserAgent has_any (UserAgentString) or HttpUserAgent matches regex UARegex
-  | summarize StartTime = min(TimeGenerated), EndTime = max(TimeGenerated) by HttpUserAgent, SourceIP = SrcIpAddr, DstIpAddr, Account = SrcUsername, URL, Type
-  | extend timestamp = StartTime, AccountCustomEntity = Account, IPCustomEntity = SourceIP, UrlCustomEntity = URL
-  ),
-  (imNetworkSession
-  | where HttpUserAgent has_any (UserAgentString) or HttpUserAgent matches regex UARegex
-  | summarize StartTime = min(TimeGenerated), EndTime = max(TimeGenerated) by HttpUserAgent, SourceIP = SrcIpAddr, DstIpAddr, Account = SrcUsername, Type, Url
-  | extend timestamp = StartTime, AccountCustomEntity = Account, IPCustomEntity = SourceIP, UrlCustomEntity = Url
```

Comment:
@oshezaf  - I removed the URL property, since it was not part of the imNetworkSession schema.
Maybe I accidently used an old schema?